tests/monitoring/monitoring.go

```diff
@@ -281,6 +290,79 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", func() {
 		})
 	})
 
+	Context("VM migration metrics", func() {
+		var nodes *k8sv1.NodeList
+
+		BeforeEach(func() {
+			scales = make(map[string]*autoscalingv1.Scale, 1)
+			backupScale(virtOperator.deploymentName)
```

Comment:
Can you please elaborate on the reason you need to scale the virt-operator in this scenario?

Reply:
Removed

Reply:
Removed